adafruit_mpu6050.py

```diff
@@ -388,3 +389,13 @@ def clock_source(self, value: int) -> None:
                 "clock_source must be ClockSource value, integer from 0 - 7."
             )
         self._clksel = value
+
+    def read_whole_fifo(self):
+        """Return raw FIFO bytes"""
+        # This code must be fast to ensure samples are contiguous
+        count = self.fifo_count
+        buf = bytearray(count)
```

Comment:
```suggestion
        buf = bytearray(count + 1)
```

I think you need space for one more byte because you use one for the register address.

Reply:
Since I set in_start to zero, my expectation was that the register address at the first byte would simply be overwritten.  Am I misunderstanding how writeto_then_readfrom works?

Reply:
Yup! I missed that. I think you are right.